templates/site/about.html

```diff
@@ -10,7 +10,56 @@
         {% set intro_content = "Take a look behind the scenes at how Marble is run" %}
         {% set git_hub_link = "https://github.com/DACCS-Climate/DACCS-executive-committee" %}
         {% set content %}
-            {% include "partials/git-doc-links.html" %}
+            {%  set table_data=[
+            {"name": "David Huard", "organization": "Ouranos", "org-profile": "https://www.ouranos.ca/en/equipe#david-huard-408", "org": true, "github": false, "orcid": false, "researchgate": false, "linkedin": false},
```

Comment:
This data structure can be simplified and made more extensible so that we can use it with any link:

```json
{"name": "Francis Charette Migneault", 
 "organization": "Ouranos",
 "links": [
    {"href": "https://orcid.org/0000-0003-4862-3349",
     "alt": "orcid",
     "icon-class": "fa-orcid"},
    {"href": "https://www.researchgate.net/profile/Francis-Charette-Migneault",
     "alt": "ResearchGate",
     "icon-class": "fa-researchgate"}
  ]
}
```

That way you don't have to include all the redundant true/false values and you can easily add more icons later without changing the html code for each one.

Reply:
Made the changes